code/joint_model_global.py

```diff
@@ -72,6 +72,8 @@ def predict(self, model, data, args, test=False, gold=False, model_r=None):
             rel_label, fts, rel_idxs, doc_id, pairs, lidx_start, lidx_end, ridx_start, ridx_end, none_rel  \
                 = self.construct_relations(prob_e, lengths, rels, list(doc_id), poss, gold=False, 
                                            ent_thresh=args.ent_thresh, rel_thresh=args.rel_thresh, ent_keys=ent_keys, test=test)
+            if len(rel_label) == 0:
+                continue
```

Comment:
I'm not sure if this is a good idea, but skipping calling `model` on line 88 when these variables are empty to prevent a crash.